frontend/src/store/reducers/trace.ts

```diff
@@ -71,7 +73,6 @@ export const traceReducer = (
 				selectedService,
 				selectedTags,
 				spansList,
-				selectedKind,
```

Comment:
why we are removing this one ?

Reply:
We are already destructuring the initial state values [here](https://github.com/SigNoz/signoz/blob/f9c214bd534740a0b90c8464f75440e2ce029b19/frontend/src/store/reducers/trace.ts#L80) which runs when the initial trace values are loaded. 
Adding the `selectedKind` value again here of value `spanKindList.find((spanKind) => spanKind.label === 'SERVER')?.value || ''` doesn't make sense, right ?

Reply:
This value can controlled from the url too.. [ref](https://github.com/SigNoz/signoz/blob/f9c214bd534740a0b90c8464f75440e2ce029b19/frontend/src/store/actions/trace/getInitialData.ts#L168) at that we need to handle that also

Reply:
Oh yup! Will fix that.